test/metabase/api/notification_test.clj

```diff
@@ -0,0 +1,63 @@
+(ns metabase.api.notification-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.notification.test-util :as notification.tu]
+   [metabase.sync.sync-metadata]
+   [metabase.test :as mt]
+   [toucan2.core :as t2]))
+
+(deftest get-notication-card-test
+  (mt/with-temp [:model/Channel {chn-id :id} notification.tu/default-can-connect-channel
+                 :model/ChannelTemplate {tmpl-id :id} notification.tu/channel-template-email-with-handlebars-body]
+    (notification.tu/with-card-notification
+      [notification {:card              {:dataset_query (mt/mbql-query users)}
+                     :notification_card {:creator_id (mt/user->id :crowberto)}
+                     :subscriptions     [{:type          :notification-subscription/cron
+                                          :cron_schedule "0 0 0 * * ?"}
+                                         {:type          :notification-subscription/cron
+                                          :cron_schedule "1 1 1 * * ?"}]
+                     :handlers          [{:channel_type notification.tu/test-channel-type
+                                          :channel_id   chn-id
+                                          :active       false}
+                                         {:channel_type :channel/email
+                                          :recipients   [{:type    :notification-recipient/user
+                                                          :user_id (mt/user->id :crowberto)}]
+                                          :template_id  tmpl-id}]}]
+      (let [notification-id (:id notification)]
+        (testing "hydrate creator, payload, subscriptions, handlers"
+          (is (=? {:id            notification-id
+                   :creator_id    (mt/user->id :crowberto)
+                   :creator       {:email "crowberto@metabase.com"}
+                   :payload_type  "notification/card"
+                   :payload       {:card_id (-> notification :payload :card_id)}
+                   :subscriptions [{:notification_id notification-id
+                                    :type            "notification-subscription/cron"
+                                    :cron_schedule   "0 0 0 * * ?"}
```

Comment:
It feels a bit weird to use cron strings for the read / write interface. Is this something we are sure about?

My bias is towards exposing something more limited.

Reply:
we used to abstract it in `pulse_channel` and I found it limiting, and we plan to expose a granular schedule soon and I think we'll probably let users write cron expression in advanced mode.